### PR TITLE
Fixing NPEs

### PR DESCRIPTION
**Pull request by cschroed-usgs**

### CODE REVIEW

File: src/main/webapp/js/views/MapFilterView.js [hunk 1]
```diff
@@ -155,7 +155,7 @@ define([
 			var waterShed = model.get('waterShed'), // String
 				states = model.get('state'); // Array
 		
-			if (waterShed || states.length > 0) {
+			if (waterShed || !_.isEmpty(states)) {
```
Comment (cschroed-usgs) [inline src/main/webapp/js/views/MapFilterView.js]: Was getting a few NPEs. @isuftin , check out this underscore method I found to solve it -- it's null-safe! I'm gonna use this urrywhere!

Comment (isuftin) [inline src/main/webapp/js/views/MapFilterView.js]: A that's pretty sweet!

Comment (isuftin) [inline src/main/webapp/js/views/MapFilterView.js]: We may also want to start looking at Lo-Dash as a drop-in-replacement for Underscore. 

More, better functionality. API compatible with Underscore. Better benchmarks. Semantic versioning. 

https://lodash.com/

Comment (cschroed-usgs) [inline src/main/webapp/js/views/MapFilterView.js]: Nice! Lodash does lazy evaluation too!

**Pull request merged**